src/Elastic.Apm/Api/Request.cs

```diff
@@ -27,11 +27,6 @@ public class Request
 		/// </summary>
 		public Dictionary<string, string> Headers { get; set; }
 
-		/// <summary>
```

Comment:
Breaking?

Reply:
Yep, will document this in our release notes.